tests/feature/TransferTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Transfer;
+use App\User;
+use App\Wallet;
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class TransferTest extends TestCase
+{
+    use DatabaseMigrations, WithoutMiddleware;
+
+    /**
+     * A send transfer test.
+     *
+     * @return void
+     */
+    public function testSendTransfer()
+    {
+        $this->seed();
+
+        $user = User::find(1);
+        $this->be($user);
+
+        $this->post('api/transfer/user/send', [
+            'receiver'  => [
+                'phone_number' => '15143338888',
+            ],
+            'amount' => 1.21,
+            'wallet_id' => 1,
+            'message' => 'Hello you are a friendly person'
+        ])->assertSee('data');
+
+        $this->assertNotNull(Transfer::find(1));
+    }
+
+    /**
+     * Receive a transfer test.
+     *
+     * @return @void
+     */
+    public function testReceiveTransfer()
+    {
+        $this->seed();
+
+        $transfer = Transfer::create([
+            'sender_wallet_id' => 1,
+            'amount' => '1000', //10 canadian dollars
+            'status' => 'pending',
```

Comment:
There should be some inline comments if the function is complex, or a variable is not obvious.